### PR TITLE
Add vue-clickaway@2.1.0 w/ git auto-update

### LINKED ISSUE #8875: [Request] Add vue-clickaway
**Library name:** vue-clickaway
**Git repository url:** https://github.com/simplesmiler/vue-clickaway
**npm package url(optional):** https://www.npmjs.com/package/vue-clickaway
**License(s):** MIT
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #8875 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/simplesmiler/vue-clickaway
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/vue-clickaway
 * License and its reference: MIT, [ref](https://github.com/simplesmiler/vue-clickaway/blob/master/LICENSE)
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 3
   - Count of stars: 134
   - Count of forks: 8
 * NPM download stats (optional):
   - Downloads in the last day: 26
   - Downloads in the last week: 1000
   - Downloads in the last month: 3540

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8875, cc @simplesmiler

### CODE REVIEW

**Comment by qwe19272375** [comment]
@pvnr0082t please help me review this PR, thank you!

File: ajax/libs/vue-clickaway/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vue-clickaway",
+  "description": "Reusable clickaway directive for reusable Vue.js components",
+  "version": "2.1.0",
+  "filename": "vue-clickaway.min.js",
+  "author": {
+    "name": "Denis Karabaza",
+    "email": "denis.karabaza@gmail.com"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/vue-clickaway/package.json]: It's better to be the same as the upstream. https://github.com/simplesmiler/vue-clickaway/blob/2.1.0/package.json#L5

File: ajax/libs/vue-clickaway/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "vue",
+    "vue-directive",
+    "vuejs"
+  ],
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/simplesmiler/vue-clickaway.git"
+  },
+  "npmName": "vue-clickaway",
```
Comment (pvnr0082t) [inline ajax/libs/vue-clickaway/package.json]: There are 15 releases on GitHub but 13 on NPM. Any reason to use npm auto-update config?

**Comment by PeterDaveHello** [comment]
Looks like we don't need the file, I'll remove it.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern  @cdnjs/intern2  @cdnjs/intern3 
please be careful about the libs partially minified, something it means the file without minify is something we don't need.

File: ajax/libs/vue-clickaway/2.1.0/vue-clickaway.common.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var Vue = require('vue');
```
Comment (PeterDaveHello) [inline ajax/libs/vue-clickaway/2.1.0/vue-clickaway.common.js]: This isn't browser i think

**Comment by qwe19272375** [comment]
thanks!

**Pull request merged**